massgov/openmass
# DP-22714: Enhance search feature for collections and data topics

**Description:**
Changed the keyword filter for Data Listings and Collections to combine multiple fields.


**Jira:** (Skip unless you are MA staff)
DP-22714


**To Test:**
- Go to /collections/qagtest.
- Verify the page matches Prod.
- Test the keyword search with words from the title, short description, term name, and term description.
- Go to /collections/qagtest/topic1.
- Verify the page matches Prod.
- Test the keyword search with words from the title, short description, term name, and term description.
- Go to /data-listing/all.
- Verify the page matches Prod.
- Test the keyword search with words from the title, short description, term name, and term description.
- Go to /data-listing/topic/energy-and-environment.
- Verify the page matches Prod.
- Test the keyword search with words from the title, short description, term name, and term description.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
